static/app/components/modals/inviteMembersModal/inviteMembersModalview.tsx

```diff
@@ -76,18 +75,29 @@ export default function InviteMembersModalView({
     </Alert>
   ) : null;
 
+  const userEmails = member?.user?.emails;
+  const isVerified = userEmails ? userEmails.some(element => element.is_verified) : false;
+
+  const isSuperUser = isActiveSuperuser();
+
+  const disableInputs = sendingInvites || complete || (!isVerified && !isSuperUser);
+
   return (
     <Fragment>
       {errorAlert}
       <Heading>{t('Invite New Members')}</Heading>
-      {willInvite ? (
-        <Subtext>{t('Invite new members by email to join your organization.')}</Subtext>
-      ) : (
+      {!isVerified && !isSuperUser ? (
+        <Alert type="warning" showIcon>
+          {t('Please verify your email before inviting other users.')}
```

Comment:
we have some very old users with unverified email so they won't have verify email in their inbox. We should add a button that would trigger that email again here in the flow.

Should we make this required? It seems to me that we should have emails verified and this would be a good way to enforce it.